packages/former/src/components/FormContent.vue

```diff
@@ -14,13 +14,29 @@
       </button>
     </div>
   </div>
+  <FormFieldSelector v-if="isFormFieldSelectorOpen">
+    <div ref="formFieldSelector" class="flex flex-col">
+      <button
+        v-for="type in availableFieldTypes"
+        :key="type"
+        :label="`select ${type} as type for the new input field`"
+        class="relative m-2 flex flex-col items-center rounded-xl border border-gray-400 bg-gray-200 p-1 hover:bg-gray-300"
```

Comment:
`relative` seems unnecessary. Haven't checked the other classes.

Reply:
@anbraten I removed the generic button component and used the button type FormKit component so i don't need to use the class anymore.